jobs/search-alert/search_alert/main.py

```diff
@@ -0,0 +1,337 @@
+            job = client.load_table_from_dataframe(abnormality_data, 'mozdata.analysis.desktop_search_alert_records', job_config = job_config)
+            job.result()
+
+    # Append to 'mozdata.analysis.desktop_search_alert_latest_daily' daily the latest abnormality date so we can properly trigger the Looker alert
+    # In Looker, time series alert check the last 2 rows in the data table (so we need to append daily even there is no abnormality o/t the alert won't work
+    query_statement = """
+        WITH
+        no_holiday_update AS (
+        SELECT
+            @submission_date AS asof,
+            "No" AS is_holiday,
+            DATE(MAX(submission_date)) AS latest_abnormality_date,
+            MAX(latest_abnormality_in_days) AS latest_abnormality_date_int
+        FROM
+            `mozdata.analysis.desktop_search_alert_records`
+        WHERE
+            is_holiday IS FALSE
+        GROUP BY
+            1,
+            2 ),
+        holiday_update AS (
+        SELECT
+            @submission_date AS asof,
+            "Yes" AS is_holiday,
+            DATE(MAX(submission_date)) AS latest_abnormality_date,
+            MAX(latest_abnormality_in_days) AS latest_abnormality_date_int
+        FROM
+            `mozdata.analysis.desktop_search_alert_records`
+        WHERE
+            is_holiday IS TRUE
+        GROUP BY
+            1,
+            2 ),
+        all_update AS (
+        SELECT
+            @submission_date AS asof,
+            "All" AS is_holiday,
+            DATE(MAX(submission_date)) AS latest_abnormality_date,
+            MAX(latest_abnormality_in_days) AS latest_abnormality_date_int
+        FROM
+            `mozdata.analysis.desktop_search_alert_records`
+        GROUP BY
+            1,
+            2 )
+        SELECT
+        asof,
+        is_holiday,
+        latest_abnormality_date,
+        latest_abnormality_date_int
+        FROM
+        no_holiday_update
+        UNION ALL (
+        SELECT
+            asof,
+            is_holiday,
+            latest_abnormality_date,
+            latest_abnormality_date_int
+        FROM
+            holiday_update)
+        UNION ALL (
+        SELECT
+            asof,
+            is_holiday,
+            latest_abnormality_date,
+            latest_abnormality_date_int
+        FROM
+            all_update)
+    """
+
+    query_job = client.query(query_statement, job_config=job_config)
+    print("Running query:", query_job.job_id)
+    latest_alert_data = (query_job.result().to_dataframe())
+    
+    if dry_run:
+        print("Dry-run mode, will not write to 'mozdata.analysis.desktop_search_alert_latest_daily'")
+    else:
+        print("Updating 'mozdata.analysis.desktop_search_alert_latest_daily'")
+        job_config = bigquery.LoadJobConfig(write_disposition = 'WRITE_APPEND')
+        job = client.load_table_from_dataframe(latest_alert_data, 'mozdata.analysis.desktop_search_alert_latest_daily', job_config = job_config)
+        job.result()
```

Comment:
is the value of `job.result()` printed or logged anywhere?

Reply:
Nope, we need this to block and wait for completing the query.

Reply:
ah ok, gotcha. I'd maybe add a quick log message just before that line to let the user that we're waiting for the bq operation to be completed? Just in case we have to debug this using logs.